notification/data.js

```diff
@@ -10,8 +10,11 @@ const localSoftwareList =
           "alternatives":[
              {
                 "name":"Jitsi Meet",
-                "url":"https://jitsi.org/jitsi-meet/"
+                "url":"https://jitsi.org/jitsi-meet/",
+                "name1":"Kanboard",
```

Comment:
Hi @Enkelena, instead of using name1 and url1, we can add entries to the array, like this:

```
[
              {
                 "name": "Jitsi Meet",
                 "url": "https://jitsi.org/jitsi-meet/"
              },
              {
                 "name": "Kanboard",
                 "url": "https://kanboard.org/"
              }
]
```

and use [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) to iterate the array and show the alternatives in normal view : normalView.js.